Header header

sensor_msgs/Image image_
float32[] histogram_

uint32 most_active_id_

uint32[] cell_id_
float64[] cell_rate_
